exercises/concept/string-vs-str/src/lib.rs

```diff
@@ -9,16 +9,16 @@ pub struct CSVRecordBuilder {
 impl CSVRecordBuilder {
     // Create a new builder
     pub fn new() -> Self {
-        unimplemented!()
+        unimplemented!("implement the `CSVRecordBuilder::new` method")
     }
 
     /// Adds an item to the list separated by a space and a comma.
     pub fn add(&mut self, val: &str) {
-        unimplemented!()
+        unimplemented!("implement the `CSVRecordBuilder::add` method")
```

Comment:
```suggestion
        unimplemented!("implement the `CSVRecordBuilder::add` method, adding {}", val)
```

This should quiet a lint about unused variables.